js/components/portal-dashboard/external-link-answer.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+interface IProps {
+  answer: any;
+}
+
+export default class ExternalLinkAnswer extends React.PureComponent <IProps>{
+  render() {
+    const { answer } = this.props;
+    const linkAnswer = answer && answer.get('answer');
+    return (
+      <div><a href={linkAnswer} target="_blank">View work</a></div>
```

Comment:
This came up in another project.  For safety, we might want to add the rel attribute too:
https://web.dev/external-anchors-use-rel-noopener/